cmd/commands/step.go

```diff
@@ -0,0 +1,177 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/gosimple/slug"
+	"github.com/inngest/inngest/inngest"
+	"github.com/inngest/inngest/pkg/cli"
+	"github.com/inngest/inngest/pkg/cli/form"
+	"github.com/inngest/inngest/pkg/cli/form/formutil"
+	"github.com/inngest/inngest/pkg/function"
+	"github.com/inngest/inngest/pkg/scaffold"
+	"github.com/spf13/cobra"
+)
+
+var ()
+
+func NewCmdSteps() *cobra.Command {
+	root := &cobra.Command{
+		Use:    "step",
+		Short:  "",
+		Hidden: true,
+	}
+
+	add := &cobra.Command{
+		Use:   "add",
+		Short: "Adds a step to the current function",
+		Run:   addStep,
+	}
+
+	root.AddCommand(add)
+	return root
+}
+
+type stepModel struct {
+	Name     string
+	Language string
+}
+
+func addStep(cmd *cobra.Command, args []string) {
+	ctx := cmd.Context()
+	model := &stepModel{}
+
+	fn, err := function.Load(ctx, ".")
+	if err != nil {
+		fmt.Println(cli.RenderError(err.Error()))
```

Comment:
Should this be prefixed with something e.g. "_Failed to load funtion:_" so that we get context if there's an internalish error?

Reply:
Load takes care of human errors for us.